integration/mediation-tests/tests-other/src/test/resources/artifacts/ESB/server/repository/deployment/server/synapse-configs/default/endpoints/StockQuote_9000_EP.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need StockQuote_9000_EP.xml? Seems it hasn't been used anywhere.

Reply:
yes. Default pack, there is no endpoint directory, we need to copy the endpoint directory into the pack. However git does not allow adding empty directory, hence dummy EP added